### ISSUE TITLE
Dice loss = 0.0000

### ISSUE THREAD

**Issue by leetesua** [imatge-upc/liverseg-2017-nipsws#13]
Hi, another small question:  the dice loss during training equals to zero in many cases. Is that normal? If not, what may be the reason? Thanks in advance.
